app/models/policy.rb

```diff
@@ -309,7 +309,7 @@ def self.search_hash(s_str)
     s_rex = Regexp.new(Regexp.escape(clean_str), true)
     {
       "$or" => [
-        {"eg_id" => s_rex},
+        {"hbx_enrollment_ids" => s_rex},
```

Comment:
is this intentional ? and what is this change for ?
Also, `hbx_enrollment_ids` is an array not a string

Reply:
change is intentional and it allows searching EA enrollment group Id which is a chain of hbx_enrollment_id in the policy finder search bar.